layouts/partials/feedback.html

```diff
@@ -9,7 +9,7 @@ <h2>{{ T "feedback_heading" }}</h2>
       <a target="_blank" rel="noopener"
         href="https://stackoverflow.com/questions/tagged/kubernetes">
         Stack Overflow</a>.
-      {{ T "layouts_docs_partials_feedback_issue" }}
+      {{ T "layouts_docs_partials_feedback_issue" | safeHTML }}
```

Comment:
I'd use `markdownify` here, and have the text to localize be Markdown, not raw HTML.

Reply:
Sure, I'll do that